applications/FluidDynamicsApplication/python_scripts/cfd_function_process.py

```diff
@@ -0,0 +1,158 @@
+# Importing the Kratos Library
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+
+
+class CFDFunctions:
+    @staticmethod
+    def GetFunction(params):
```

Comment:
`GetFunction` is not a process. It is a wrapper for cfd utility methods. These are wrapper functions to be used in the json settings. All of them are standalone functions. As an example, this way it enables me to use reaction based y+ (for no slip) and wall law based y+(for slip with wall functions) calculations in the same simulation, and distribute them to nodes appropriately. (Still the common node between no slip and slip will have a problem). 

This also makes having new cfd methods easily extensible to json settings using the same `cfd_function_process.py`. No need to have different python processes for different cfd utility methods.

Reply:
THB I don't love the idea of implementing an additional concept in parallel to the processes. I.e. this could easily be done with separate processes (for which several py-scripts are needed ofc).

Is the only reason to do it as you propose bcs you don't want the additional py-scripts? Or would you have additional overhead?

To me the current script (`cfd_function_process.py`) is completely unreadable :/

If we really want to go this way then why not what is done in the HDF or the statistics app? There it is cleaner IMO